packages/web/src/components/gcds-card/stories/gcds-card.stories.tsx

```diff
@@ -123,3 +139,55 @@ Default.args = {
   imgAlt: '',
   footer: '',
 };
+
+export const Link = Template.bind({});
+Link.args = {
+  cardTitle: 'Card title link',
+  href: '#',
+  type: 'link',
+  titleElement: 'a',
+  tag: '',
+  description: 'Description or supporting text relating to the headline. Longer text will be truncated with ...',
+  imgSrc: '',
+  imgAlt: '',
+  footer: '',
+};
+
+export const Action = Template.bind({});
+Action.args = {
+  cardTitle: 'Card title link',
+  href: '#',
+  type: 'action',
+  titleElement: 'a',
+  tag: '',
+  description: 'Description or supporting text relating to the headline. Longer text will be truncated with ...',
+  imgSrc: '',
+  imgAlt: '',
+  footer: 'Action button/link',
```

Comment:
Could we potentially show a button or link here? Just to really drive home the point haha